ts/components/game/Utils.ts

```diff
@@ -0,0 +1,14 @@
+import * as Immutable from "immutable";
+import Entity from "../../Entity";
+
+export function findCreator(creatorId:number, entities:Immutable.Map<number, Immutable.Map<number, Entity>>) {
```

Comment:
Have you considered making this a method of Entity instead, a la `entity.getCreator()`? Other than that, gtm

Reply:
Find the creator entity requires the list of all entities, are you suggesting something like

Reply:
I thought you could get the game object from the Entity but apparently not. So what I suggest is instead passing the game as argument.